exporter/exporterhelper/common.go

```diff
@@ -76,3 +83,35 @@ func errToStatus(err error) trace.Status {
 	}
 	return okStatus
 }
+
+// GrpcSettingsToDialOptions maps configgrpc.GRPCSettings to a slice of dial options for gRPC
+func GrpcSettingsToDialOptions(settings configgrpc.GRPCSettings) ([]grpc.DialOption, error) {
```

Comment:
I think this will be better in a separate package: some types depending on `exporterhelper` do not need to depend on gRPC related stuff. It seems to make more sense to have it as part of `configgrpc` (or perhaps as a new package).